### ISSUE TITLE
Request: ability to set the start/end time for the raid

### ISSUE THREAD

**Issue by jrgensol** [SilphRoad/reports#13]
It would be nice to be able to set the start/end time for a raid. So that people out of range of the raid in game can still know when it starts/ends. In case they don't have time to be there at the same time as you but can still see if they still have time to go before it ends, or if they don't have time to wait for you they can see if it starts early enough for them to still reach their other plans. Now all the info they have is that at a point in time.

**Comment by adrazich** [comment]
This is the part that confuses me the most. When you create the raid you say when you will be attending. It then uses that time as the start time for the raid with a countdown until that time. Once the countdown is finished it switches to "Now!" which gives me the assumption that the raid has begun. However, that really isn't the case. That was when the first person said they would be there. 

Other trainers can then say, "I'll be attending," and it increases the count but it doesn't let them say when they plan to be there as well.

I think it would be far less confusing if the first user inserted the start time. You can then determine the end time based on that. Then, each person who says they are attending can say their estimated time of arrival. Then, you would have an understanding of when the raid actually begins, expires, how many trainers are showing up and their time of arrival.

**Comment by Wolfeitz** [comment]
Yes - there needs to be an actual Raid Countdown - the time left that the boss will be there.

With the legendaries we were seeing multiple groups all raiding the same mon.
Allow people to put their ETA, then count that down. 

Then there needs to be a (multiple) Raid starting in XX countdown.

If you do that, then it would be possible to create alerts letting people know that someone will be there in about 2 mins if they can just hold off starting.  Or that everyone is there already, go ahead and start.

**Comment by fortepc** [comment]
Agreed, more detailed RSVP system should be implimented.
